JesusFilm/core
# feat: eng 705 searchbar dropdown tabs

# Description

### Issue

_Provide a brief summary of why this task is needed_

[Link to Linear Todo](https://linear.app/jesus-film-project/issue/ENG-705/searchbar-dropdown-tabs)

### Solution

<img width="1012" alt="image" src="https://github.com/user-attachments/assets/de56d1bd-3be7-418d-83a7-7f89f077d7b0">

### External Changes
I changed the spacing of the content within the dropdown

### How to test
* User should be able to view search suggestions by default
* User should be able to navigate to language tab
* User should still be able to apply suggestions
* User should still be able to apply language filters
